File: Semestru 4/MPP/lab/java/lab3/Model/src/main/java/org/example/domain/Event.java
package org.example.domain;

public class Event extends Entity<Integer>{
    public String Name;
    public int MinAge;
    public int MaxAge;

    public Event (String Name, int MinAge, int MaxAge) {
        this.Name = Name;
        this.MinAge = MinAge;
        this.MaxAge = MaxAge;
    }

    public String GetName() {
        return Name;
    }

    public int GetMinAge() {
        return this.MinAge;
    }

    public int GetMaxAge() {
        return this.MaxAge;
    }

    @Override
    public String toString() {
        return this.Name + ' ' + this.MinAge + ' ' + this.MaxAge;
    }
}

File: Semestru 4/MPP/lab/.net/lab6/restAPI/restAPI/Model/Domain/Event.cs
namespace restAPI.Model.Domain
{
	public class Event : Entity<int>
	{
		public string Name { get; set; }
		public int MinAge { get; set; }
		public int MaxAge { get; set; }

		public Event(string name, int minAge, int maxAge )
		{
			Name = name;
			MinAge = minAge;
			MaxAge = maxAge;
		}

		public Event() { }
	}
}
